### DISCUSSION TITLE
Add Portals to CK Map Tool

### DISCUSSION THREAD

**Discussion by bigdaddyseany** [Ceddini/CoreKeeperMapTool#25] [Feature Requests]
Hi, great tool!  I was not seeing an option to view Portals I've played onto the world, is this a feature that can be added?

**Comment by Ceddini** [comment]
Can't really do it, since I only display the "minimap" of the player and that doesn't contain any info on placed items, except the color of the pixel. Portal doesn't have a pixel color though, so I can't really highlight it.
